compiler/aarch64/codegen/PseudoRegisterEnum.hpp

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2018 IBM Corp. and others
```

Comment:
I think we need to update these copyright dates to "2018, 2018" as this is a completely new file. Similarly elsewhere.

Reply:
Done: 3d92f53441d758ed3d79bdfa0e9e86f4d07968d9